package.json

```diff
@@ -68,9 +68,9 @@
     "babel-standalone": "^6.4.4",
     "codemirror": "^5.12.0",
     "css-loader": "~0.9.0",
-    "react-addons-test-utils": "^15.0.1",
-    "react": "15.0.1",
-    "react-dom": "^15.0.1",
+    "react-addons-test-utils": "^15.0.0 || ^0.14.7",
+    "react": "15.0.1 || ^0.14.7",
```

Comment:
Shouldn't this be `"react": "^15.0.1 || ^0.14.7",` to keep in-step with `react-dom`, and `react-addons-test-utils`?